src/main/kotlin/no/nav/syfo/personstatus/application/OppfolgingstilfelleService.kt

```diff
@@ -3,16 +3,28 @@ package no.nav.syfo.personstatus.application
 import no.nav.syfo.oppfolgingstilfelle.domain.Oppfolgingstilfelle
 import no.nav.syfo.personstatus.domain.PersonIdent
 import no.nav.syfo.personstatus.domain.PersonOversiktStatus
+import no.nav.syfo.personstatus.infrastructure.clients.pdl.model.fodselsdato
+import no.nav.syfo.personstatus.infrastructure.clients.pdl.model.fullName
+import org.slf4j.LoggerFactory
 import java.time.temporal.ChronoUnit
 
 class OppfolgingstilfelleService(
+    private val pdlClient: IPdlClient,
     private val personOversiktStatusRepository: IPersonOversiktStatusRepository,
 ) {
-
-    fun upsertPersonOversiktStatus(personStatus: PersonOversiktStatus, newPersonOppfolgingsTilfelle: Oppfolgingstilfelle) {
+    suspend fun upsertPersonOversiktStatus(personStatus: PersonOversiktStatus, newPersonOppfolgingsTilfelle: Oppfolgingstilfelle) {
         val existingPerson: PersonOversiktStatus? = personOversiktStatusRepository.getPersonOversiktStatus(PersonIdent(personStatus.fnr))
         if (existingPerson == null) {
-            personOversiktStatusRepository.createPersonOversiktStatus(personStatus)
+            pdlClient.getPerson(PersonIdent(personStatus.fnr))
+                .map {
+                    val editedPersonStatues =
+                        personStatus.updatePersonDetails(navn = it.fullName(), fodselsdato = it.fodselsdato())
+                    log.info("Got person from PDL: ${editedPersonStatues.navn} with fnr: ${editedPersonStatues.fnr}")
```

Comment:
Fjerne denne linja før prodsetting.

Reply:
Yup! Var for testing i dev for å få testet å konsumere oppfolgingstilfelle. Tatt bort 😊